index.html

```diff
@@ -62,6 +62,142 @@ <h2 class="second-header">I'm a Software Developer</h2>
             <div class="main-image"></div>
         </section>
     </main>
+
+    <section class="work-section">
+        <h2 class="second-title">My recent Works</h2>
+        <hr class="line_below_title">
+        <div class="card-container">
+            <div class="main_card">
+                <img class="image_from_main_card" src="./assets/images/cardImages/mainCard.png" alt="Work 1">
+                <p class="main-card-title">Multi-Post Stories</p>
+                <p class="main-card-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.
+                </p>
+                <ul class="list-ul">
+                    <li class="list-li">CSS</li>
+                    <li>HTML</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="button-from-main-card" type="button">See project</button>
+            </div>
+        </div>
+
+        <div class="works-container">
+            <div class="work-card">
+                <p class="work-cards-title">Profesional Art Printing Data</p>
```

Comment:
- [ ] As mentioned earlier, this project card title should use any of the `hx` tags.